jerry-debugger/src/cli/cli.ts

```diff
@@ -45,17 +48,24 @@ export function getOptionsFromArgs(argv: Array<string>) {
   return {
     ...getHostAndPort(target),
     verbose: args['verbose'],
-    jsfile: args._[0],
+    address: args._[0] || '',
+    jsfile: args['client-source'] || 'untitled.js',
```

Comment:
I'd like to stick with the node.js CLI semantics as much as possible because this will make integrating with existing IDEs/editors much easier:

```
$ node --help
Usage: node [options] [ -e script | script.js | - ] [arguments]
       node inspect script.js [arguments]

Options:
  -v, --version              print Node.js version
  -e, --eval script          evaluate script
  -p, --print                evaluate script and print result
  -c, --check                syntax check script without executing
  -i, --interactive          always enter the REPL even if stdin
                             does not appear to be a terminal
  -r, --require              module to preload (option can be repeated)
  -                          script read from stdin (default; interactive mode if a tty)
  --inspect[=[host:]port]    activate inspector on host:port
                             (default: 127.0.0.1:9229)
  --inspect-brk[=[host:]port]
                             activate inspector on host:port
                             and break at start of user script
  --inspect-port=[host:]port
                             set host:port for inspector
... etc ...
```

Reply:
For new, JerryScript specific arugments, let's prefix with `--jerry-...` to avoid conflicts. How does that sound?